owncloudApp/src/main/res/values-ko/strings.xml

```diff
@@ -236,8 +236,7 @@
   <string name="auth_wtf_reenter_URL">예상하지 못한 상태, 서버 주소를 다시 입력하십시오</string>
   <string name="auth_expired_oauth_token_toast">인증이 만료되었습니다. 다시 인증하십시오</string>
   <string name="auth_expired_basic_auth_toast">현재 암호를 입력하십시오</string>
-  <string name="auth_expired_saml_sso_token_toast">세션이 만료되었습니다. 다시 접속하십시오</string>
-  <string name="auth_connecting_auth_server">인증 서버에 연결하는 중…</string>
+    <string name="auth_connecting_auth_server">인증 서버에 연결하는 중…</string>
```

Comment:
Why is this line indented? Same as a lot of strings files.

Reply:
This is because I used the Translations Editor of Android Studio to remove the entries with `auth_expired_saml_sso_token_toast` key and it seems that editor indented the `auth_connecting_auth_server` string that way.  Otherwise, I would have to go through a ton of strings.xml files and remove the entries manually.
Now that I have had a look at it, 4 spaces is the proper indenting so I would keep it as it is.